From b17b52747b1b70d9ac6cfe0647b77f7d84efd771 Mon Sep 17 00:00:00 2001
From: Aleksander Morgado <aleksander@aleksander.es>
Date: Fri, 26 Dec 2014 16:36:32 +0100
Subject: [PATCH] iface-modem-location: don't print location info to syslog
 with info level

Use debug level, which has to be explicitly requested by the user.

https://bugs.freedesktop.org/show_bug.cgi?id=87498
(cherry picked from commit 0dd707a2793c28369991377a0e09e00d3a09636f)
---
 src/mm-iface-modem-location.c | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/src/mm-iface-modem-location.c b/src/mm-iface-modem-location.c
index 57d5f2c..f97789e 100644
--- a/src/mm-iface-modem-location.c
+++ b/src/mm-iface-modem-location.c
@@ -216,8 +216,8 @@ notify_gps_location_update (MMIfaceModemLocation *self,
     const gchar *dbus_path;
 
     dbus_path = g_dbus_object_get_object_path (G_DBUS_OBJECT (self));
-    mm_info ("Modem %s: GPS location updated",
-             dbus_path);
+    mm_dbg ("Modem %s: GPS location updated",
+            dbus_path);
 
     /* We only update the property if we are supposed to signal
      * location */
@@ -286,13 +286,13 @@ notify_3gpp_location_update (MMIfaceModemLocation *self,
     const gchar *dbus_path;
 
     dbus_path = g_dbus_object_get_object_path (G_DBUS_OBJECT (self));
-    mm_info ("Modem %s: 3GPP location updated "
-             "(MCC: '%u', MNC: '%u', Location area code: '%lX', Cell ID: '%lX')",
-             dbus_path,
-             mm_location_3gpp_get_mobile_country_code (location_3gpp),
-             mm_location_3gpp_get_mobile_network_code (location_3gpp),
-             mm_location_3gpp_get_location_area_code (location_3gpp),
-             mm_location_3gpp_get_cell_id (location_3gpp));
+    mm_dbg ("Modem %s: 3GPP location updated "
+            "(MCC: '%u', MNC: '%u', Location area code: '%lX', Cell ID: '%lX')",
+            dbus_path,
+            mm_location_3gpp_get_mobile_country_code (location_3gpp),
+            mm_location_3gpp_get_mobile_network_code (location_3gpp),
+            mm_location_3gpp_get_location_area_code (location_3gpp),
+            mm_location_3gpp_get_cell_id (location_3gpp));
 
     /* We only update the property if we are supposed to signal
      * location */
@@ -403,11 +403,11 @@ notify_cdma_bs_location_update (MMIfaceModemLocation *self,
     const gchar *dbus_path;
 
     dbus_path = g_dbus_object_get_object_path (G_DBUS_OBJECT (self));
-    mm_info ("Modem %s: CDMA BS location updated "
-             "(Longitude: '%lf', Latitude: '%lf')",
-             dbus_path,
-             mm_location_cdma_bs_get_longitude (location_cdma_bs),
-             mm_location_cdma_bs_get_latitude (location_cdma_bs));
+    mm_dbg ("Modem %s: CDMA BS location updated "
+            "(Longitude: '%lf', Latitude: '%lf')",
+            dbus_path,
+            mm_location_cdma_bs_get_longitude (location_cdma_bs),
+            mm_location_cdma_bs_get_latitude (location_cdma_bs));
 
     /* We only update the property if we are supposed to signal
      * location */
-- 
2.1.0

